pkg/skaffold/runner/runner.go

```diff
@@ -266,6 +290,14 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*v1
 		}
 	}
 
+	// Watch test configuration
+	if err := watcher.Register(
+		func() ([]string, error) { return r.TestDependencies(), nil },
+		func() { changed.needsRedeploy = true },
```

Comment:
Will this trigger a retest?

Reply:
yep, I added a call to `r.Test` [here](https://github.com/GoogleContainerTools/skaffold/pull/1013/files#diff-a6568a246e086f5b70b0a00bf56093a8R260)